vue/src/store/modules/workspace/types.d.ts

```diff
@@ -0,0 +1,3 @@
+export interface WorkspaceState {
```

Comment:
@skavt Please remove this file, we do not use types checks for vuex module data.